examples/vello_editor/src/text.rs

```diff
@@ -26,13 +29,24 @@ pub enum ActiveText<'a> {
     Selection(&'a str),
 }
 
+#[derive(Default)]
+enum ComposeState {
+    #[default]
+    None,
+    Preedit {
+        /// The location of the (uncommitted) preedit text
+        text_at: Selection,
+    },
+}
+
```

Comment:
If dead key composing is something that we for now hope will silently be converted to IME events by Winit in the future, I don't expect any other `ComposeState` to be added here. In that case this should just become `Option<PreeditState>`.

Reply:
I've removed the enum in favor of the simpler `preedit_range: Option<Range<usize>>`